ios/InstagramFirebase/CommentInputTextView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  CommentInputTextView.swift
+//  InstagramFirebase
+//
+//  Created by Brian Voong on 1/3/18.
+//  Copyright © 2018 Lets Build That App. All rights reserved.
+//
+
+import UIKit
+
+class CommentInputTextView: UITextView {
+    
+    fileprivate let placeholderLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Enter Comment"
+        label.textColor = UIColor.lightGray
+        return label
+    }()
+    
+    func showPlaceholderLabel() {
```

Comment:
Instead of function for UI components, It'll be great below closure code.
``
```suggestion
 fileprivate let personNameLabel: UILabel = {
    let label = UILabel()
    label.backgroundColor = .gray
    label.text = "Please click here."
    return label
}()
```